test/README.md

```diff
@@ -149,6 +155,8 @@ webhooks:
 
 The difference between this one and the one generate by the `kudo init --unsafe-self-signed-webhook-ca` command, (see this [method](pkg/kudoctl/kudoinit/prereq/webhook.go:163) for more information) is the usage of `webhooks[].clientConfig.url` (which points to our ngrok-tunnel) instead of `webhooks[].clientConfig.Service`.
 
+**note:** The url used by ngrok changes for each run of ngrok.  It is possible to get the current url with `curl -s localhost:4040/api/tunnels | jq '.tunnels[1].public_url'`
```

Comment:
I think it's more complicated than it should be: the tunnel only exists while the `ngrok` is running in your terminal window and it already gives you the URL. No need to get it from the API.